### DISCUSSION TITLE
How to return files with custom tools

### DISCUSSION THREAD

**Discussion by Depblu** [langgenius/dify#10095] [Help]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### 1. Is this request related to a challenge you're experiencing? Tell me about your story.

Hi,
    I want to call a custom tool within a workflow to generate a list of images, and then access the visual features in an LLM node, using the images generated by the tool. How can I achieve this? 

Thanks.

![屏幕截图_20241031_105206](https://github.com/user-attachments/assets/bf9c1209-c262-4687-bfdd-393fdf34cd3d)

**Comment by meng-tsai** [comment]
bump

**Comment by 826541814** [comment]
i have same problem
